src/components/panel/panel.js

```diff
@@ -1293,7 +1351,7 @@ MdPanelRef.prototype._animateClose = function() {
   return this._$q(function(resolve, reject) {
     animationConfig.animateClose(self._$q).then(function(){
       self.removeClass('md-panel-is-showing');
-      resolve(self);
+      return resolve(self);
```

Comment:
when using this style I think you can omit the `return`; it's only when chaining promises that you have to explicitly return.